src/Nowruz/pages/sign-up/Email/index.tsx

```diff
@@ -53,7 +53,7 @@ export const Email = () => {
                 tried();
                 navigate('/oauth/google');
               }}
-              className="flex w-full gap-4 normal-case text-base mt-4"
+              style={{ display: 'flex', gap: '12px', width: '100%', marginTop: '12px' }}

```

Comment:
why we use style instead ? is design different in sign in vs sign up ? if no do we need change sign in too ?

Reply:
@jeyem both sign in and sign up uses style